App.tsx

```diff
@@ -4,11 +4,19 @@
  *
  * @format
  */
-import React from 'react';
+
+import React, {useEffect} from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {RootNavigator} from './src/navigation/RootNavigator';
+import RNBootsplash from 'react-native-bootsplash';
 
 function App(): JSX.Element {
+  useEffect(() => {
+    RNBootsplash.hide({
```

Comment:
If you hide the splash here, then there will be a moment of blank screen before Home screen is loaded.
So you hide this in Home if you do not have any login and Home will always be your first screen. If you do have a login flow, then create a screen called Splash and hide there. Any common logics you need to handle before showing login or home screen will also be done in Splash Screen

Reply:
Yeah i got the issue we can hide this on navigator mount at NavigationContainer wrapper.